resources/views/pages/home.blade.php

```diff
@@ -2,18 +2,57 @@
 
 @section('main_content')
 
-    @include('layouts.header', ['header' => 'Rogue', 'subtitle' => 'Please log in to continue.'])
+    @include('layouts.header', ['header' => 'Rogue', 'subtitle' => 'Please log in with your DoSomething.org admin account to continue.
+'])
 
     <div class="container -padded">
         <div class="wrapper">
-            <div class="container__block -narrow">
+            <div class="container__block">
                 <p>
-                    Welcome to <strong>Rogue</strong>, our campaign activity admin tool.
+                    Welcome to <strong>Rogue</strong>, our one source of truth for all things user activity. Logging in will allow you to review and search for reportbacks.
                 </p>
 
-                <p>
+                <h4>We're Team Bleed - your friends behind building and improving Rogue!</h4>
+                <div class="container__block -half">
+                    <ul class>
+                        <li>
```

Comment:
These might work nicely as `.figure.-left.-small`s:

<img width="184" alt="screen shot 2017-11-07 at 1 50 07 pm" src="https://user-images.githubusercontent.com/583202/32511706-9557b2a0-c3c2-11e7-938f-fdef9cd7df2c.png">

Reply:
oooooo

Reply:
ooooo yes!! Love this!!

Reply:
@DFurnes updated this and updated the screenshot in the description! Great call!